### DISCUSSION TITLE
custom F-droid repo

### DISCUSSION THREAD

**Discussion by imxade** [ImranR98/Obtainium#943] [Q&A]
how to add custom F-droid repo ?

**Comment by ImranR98** [comment]
You need the repo URL and the ID or name of the app you want. Also, Obtainium will not be able to detect that the URL you provided is an F-Droid repo, so you'll need to select the source manually. 

For example, to get the Tor Browser from the Guardian Project Repo, you would use these settings:
- App Source URL: https://guardianproject.info/fdroid/repo/
- Override Source: F-Droid Third-Party Repo
- App ID or Name: Tor Browser

See https://github.com/ImranR98/Obtainium/wiki#f-droid-third-party-repo
